ietf/mailinglists/migrations/0001_initial.py

```diff
@@ -40,15 +40,15 @@ class Migration(migrations.Migration):
             },
         ),
         migrations.CreateModel(
-            name='Whitelisted',
+            name='Allowlisted',
```

Comment:
Making this change in the 0001_initial migration will have no effect on the production database. This file should be left alone to allow the migration system to work properly.

What needs to happen instead is a change to the model (and it's associated admin and resource files) and a _new_ migration that will make the change to the data.